### PR TITLE
Release: Gateway 2.8.4.8

### PR DESCRIPTION
**Pull request by lena-larionova**
### Description

Changelog, version bump, and generating kong.conf reference.

https://konghq.atlassian.net/browse/DOCU-3738
### Testing instructions

Preview link: https://deploy-preview-7136--kongdocs.netlify.app/gateway/changelog/#2848

### Checklist 

- [x] Review label added 
- [x] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

For example, if this change is for an upcoming 3.6 release, enclose your content in `{% if_version gte:3.6.x %} <content> {% endif_version %}` tags (or `if_plugin_version` tags for plugins). 

Use any of the following keys:
* `gte:<version>` - greater than or equal to a specific version
* `lte:<version>` - less than or equal to a specific version
* `eq:<version>` - exactly equal to a specific version

You can do the same for older versions.

### CODE REVIEW

File: app/gateway/changelog.md [hunk 1]
```diff
@@ -4105,6 +4105,54 @@ openid-connect
...
+  * Fixed an issue with `sync_rate` setting being used with the `redis` strategy. 
+  If the Redis connection is interrupted while `sync_rate = 0`, the plugin now accurately falls back to the `local` strategy.
+
+* [Rate Limiting](/hub/kong-inc/rate-limiting/) (`rate-limiting`), [Rate Limiting Advanced](/hub/kong-inc/rate-limiting-advanced/) (`rate-limiting-advanced`), [GraphQL Rate Limiting Advanced](/hub/kong-inc/graphql-rate-limiting-advanced/) (`graphql-rate-limiting-advanced`), and [Response Rate Limiting](/hub/kong-inc/response-ratelimiting/) (`response-ratelimiting`)
+  * Fixed an issue where any plugins using the `rate-limiting` library, when used together, 
+  would interfere with each other and fail to synchronize counter data to the central data store.
+
+### Dependencies
+
+* Bumped `lua-resty-openssl` to 1.2.1
```
Comment (outsinre) [inline app/gateway/changelog.md:4150]:
```suggestion
```

This is removed. See https://github.com/Kong/kong-ee/pull/8638/files.

File: app/gateway/changelog.md [hunk 2]
```diff
@@ -4105,6 +4105,53 @@ openid-connect
...
+* Fixed the UDP socket leak of the DNS module.
+
+#### Plugins
+
+* [LDAP Auth Advanced](/hub/kong-inc/ldap-auth-advanced/) (`ldap-auth-advanced`)
+  * Fixed some cache-related issues which caused `groups_required` to return unexpected codes after a non-200 response.
+
+* [Rate Limiting Advanced](/hub/kong-inc/rate-limiting-advanced/) (`rate-limiting-advanced`)
+  * Fixed an issue with `sync_rate` setting being used with the `redis` strategy. 
+  If the Redis connection is interrupted while `sync_rate = 0`, the plugin now accurately falls back to the `local` strategy.
```
Comment (Guaris) [inline app/gateway/changelog.md:4142]:
```suggestion
  *  Fixed an issue where the `sync_rate` setting, when being used with the `redis` strategy did not properly revert to the `local` strategy if the Redis connection was interrupted with `sync_rate = 0`.
```

**Review by Guaris — APPROVED** [review]

File: app/gateway/changelog.md (hunk 2 above)
Comment (lena-larionova) [inline app/gateway/changelog.md:4142]:
```suggestion
  * Fixed an issue where, if `sync_rate` was set to `0` and the `redis` strategy was in use, the plugin did not properly revert to the `local` strategy if the Redis connection was interrupted.
```

Comment (lena-larionova) [reply app/gateway/changelog.md]: Rephrased it a bit more and applied that.

**Pull request merged**